atoma-proxy-service/docs/openapi.yml

```diff
@@ -636,38 +636,6 @@ paths:
               schema: {}
         '500':
           description: Failed to get graphs
-  /google_oauth:
```

Comment:
Why was this removed? Doesn't seem related to this PR

Reply:
my mistake